--- include/elements/Makefile.am.install	2010-02-05 04:03:44.000000000 +0100
+++ include/elements/Makefile.am	2010-02-05 04:04:08.000000000 +0100
@@ -21,8 +21,6 @@
                      CEGUIItemListBaseProperties.h \
                      CEGUIItemListbox.h \
                      CEGUIItemListboxProperties.h \
-                     CEGUIListHeader.h \
-                     CEGUIListHeaderProperties.h \
                      CEGUIListbox.h \
                      CEGUIListboxItem.h \
                      CEGUIListboxProperties.h \
